core/ipc/rpc/kafka/src/main/java/org/opennms/core/ipc/rpc/kafka/KafkaRpcServerManager.java

```diff
@@ -315,6 +368,8 @@ private void handleRequest(RpcMessageProto rpcRequestProto, ByteString rpcConten
             setTagsOnMinion(rpcRequestProto, request, minionSpan);
             // Modules may run the execution in their own thread pool.
             CompletableFuture<RpcResponse> future = module.execute(request);
+            final Meter requestSentMeter = getMetrics().meter(MetricRegistry.name(request.getLocation(), module.getId(), RPC_COUNT));
```

Comment:
We can omit location here since all the requests will be for the same location right?

Reply:
Maybe we should rename to `requestsReceived`? From this perspective we're processing the received request.